app/retail/templates/bounties/contributor/activity.html

```diff
@@ -16,31 +16,24 @@
 {% endcomment %}
 {% load i18n static  %}
 <link rel="stylesheet" href={% static "v2/css/activity_stream.css" %}>
+<style>
+  .contributor-page .box {
```

Comment:
Could we store styles in css file?

Reply:
I did wanna do that initially but throwing the landing page stylesheet forces me to use `!important` as rules in `activity_stream.css` overwrites them and throwing it onto `activity_stream.css` causes it to be imported in places where it's not needed 😓